File: src/main/java/com/ptah/camunda/delegate/FlowADelegate.java
package com.ptah.camunda.delegate;

import org.camunda.bpm.engine.delegate.DelegateExecution;
import org.camunda.bpm.engine.delegate.JavaDelegate;
import org.springframework.stereotype.Component;

@Component("FlowADelegate")
public class FlowADelegate implements JavaDelegate {
    @Override
    public void execute(DelegateExecution execution) throws Exception {
        execution.setVariable("Hello","I am Flow A");
    }
}

File: backend/src/main/kotlin/com/gihub/jarryzhou/ptah/camunda/delegate/FlowADelegate.kt
package com.gihub.jarryzhou.ptah.camunda.delegate

import org.camunda.bpm.engine.delegate.DelegateExecution
import org.camunda.bpm.engine.delegate.JavaDelegate
import org.springframework.stereotype.Component

@Component("FlowADelegate")
class FlowADelegate : JavaDelegate {
    @Throws(Exception::class)
    override fun execute(execution: DelegateExecution) {
        execution.setVariable("Hello", "I am Flow A")
    }
}
